lib/ddtrace/profiling/scheduler.rb

```diff
@@ -12,7 +12,12 @@ class Scheduler < Worker
       include Workers::Polling
 
       DEFAULT_INTERVAL_SECONDS = 60
-      MIN_INTERVAL_SECONDS = 0
+      MINIMUM_INTERVAL_SECONDS = 0
+
+      # Profiles with duration less than this will not be reported
+      MINIMUM_FLUSH_CONTENT_SECONDS = 1
```

Comment:
just dropping in here without much context, but wanted to see if there might be a slightly clearer name for this variable? Since its a threshold duration for a profile in order to be collected, it could be called something like `THRESHOLD_PROFILE_DURATION`?

Reply:
👍  Renamed to PROFILE_DURATION_THRESHOLD_SECONDS in 85f21d1717895cc93340af0bd4ccf8cc66ab9b86